### PR TITLE
Remove kube-reserved which is no longer supported

### PR DESCRIPTION
**Pull request by mburke5678**
Remove kubeReserved from the docs. This setting is [not used with OpenShift Container Platform](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/nodes/index#nodes-nodes-resources-configuring-about_nodes-nodes-resources-configuring).

https://issues.redhat.com/browse/OCPBUGS-31542

Previews:

Scalability -> NUMA aware -> [Deploying the NUMA-aware secondary pod scheduler with manual performance settings](https://73939--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling#cnf-deploying-the-numa-aware-scheduler-with-manual-performance-settings_numa-aware) -- Removed `kube-reserved` from code example in Step 1a.
[Reserving CPUs for nodes](https://73939--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-resources-cpus#nodes-nodes-resources-cpus-reserve_nodes-nodes-resources-cpus) -- Removed `kube-reserved` from first paragraph and additional resources.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio PTAL

**Comment by mburke5678** [comment]
@sunilcio Thank you for the quick review!

File: nodes/nodes/nodes-nodes-resources-cpus.adoc [hunk 1]
```diff
@@ -19,4 +19,4 @@ include::modules/nodes-nodes-resources-cpus-reserve.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-* For more information on the `systemReserved` and `kubeReserved` parameters, see xref:../../nodes/nodes/nodes-nodes-resources-configuring.adoc#nodes-nodes-resources-configuring-about_nodes-nodes-resources-configuring[Allocating resources for nodes in an {product-title} cluster].
+* For more information on the `systemReserved` parameter, see xref:../../nodes/nodes/nodes-nodes-resources-configuring.html#nodes-nodes-resources-configuring-about_nodes-nodes-resources-configuring[Allocating resources for nodes in an {product-title} cluster].
```
Comment (gaurav-nelson) [inline nodes/nodes/nodes-nodes-resources-cpus.adoc:22]: **nitpick:** The link should stay as `.adoc` and not `.html`.

**Comment by gaurav-nelson** [comment]
Thank you @mburke5678 I've marked one nit we can merge this after that change.

File: nodes/nodes/nodes-nodes-resources-cpus.adoc (hunk 1 above)
Comment (mburke5678) [reply nodes/nodes/nodes-nodes-resources-cpus.adoc:22]: I take it back. No idea how that change happened. Thanks for catching it.

**Pull request merged**